### ISSUE TITLE
Support for Web vtt captions on video

### ISSUE THREAD

**Issue by craigS89** [mastodon/mastodon#29153]
### Pitch

Let users add a .vtt closed caption file to videos.
The HTML video player already supports this file type, so users should be able to attach it to their videos.

### Motivation

This is an important accessibility feature for the deaf or hard of hearing. As mastodon does not generate automatic closed captions like a lot of other social sites, users should have the option of being able to add their own to ensure videos have the option of being accesable.

**Comment by aardrian** [comment]
Related 2018 suggestion that also has no activity: https://github.com/mastodon/mastodon/issues/7169
